client/src/containers/pages/Home/Home.tsx

```diff
@@ -36,13 +37,15 @@ const Home: React.FC<HomeProps> = (props: HomeProps) => {
             </Link>
           </Col>
         )}
-        <Col className="gutter-row" span={12}>
-          <Link to={URL_TEAMS} className="admin-link">
-            <Button color="outline" className="btn-links">
-              Teams
-            </Button>
-          </Link>
-        </Col>
+        {ENABLE_LOCATIONS && (
```

Comment:
We also need to account for this in the tests i.e when `ENABLE_LOCATIONS` is `true` by mocking the value to `true` like it's done [here](https://github.com/OpenSRP/web/blob/import-form-config/client/src/containers/ConnectedSidebar/tests/index.test.tsx#L53)

Reply:
addresses